go/test/endtoend/reparent/utils/utils.go

```diff
@@ -762,8 +762,10 @@ func ReplicationThreadsStatus(t *testing.T, status *replicationdatapb.Status, vt
 		// and should agree with the new ones
 		require.NotEqual(t, mysql.ReplicationStateUnknown, mysql.ReplicationState(status.IoState))
 		require.NotEqual(t, mysql.ReplicationStateUnknown, mysql.ReplicationState(status.SqlState))
-		require.Equal(t, status.IoThreadRunning, mysql.ReplicationState(status.IoState) == mysql.ReplicationStateRunning || mysql.ReplicationState(status.IoState) == mysql.ReplicationStateConnecting)
-		require.Equal(t, status.SqlThreadRunning, mysql.ReplicationState(status.SqlState) == mysql.ReplicationStateRunning || mysql.ReplicationState(status.SqlState) == mysql.ReplicationStateConnecting)
+		if vttabletVersion == 14 {
```

Comment:
are they specific to 14 only or should it be >= 14

Reply:
That is specific to v14, because the fields IoThreadRunning are legacy fields removed in v15+. They are present in v14 only for compatilbilty.